src/VisualStudio/Core/Impl/Options/OptionPreviewControl.xaml.cs

```diff
@@ -19,13 +22,28 @@ internal OptionPreviewControl(IServiceProvider serviceProvider, Func<OptionSet,
         {
             InitializeComponent();
 
-            _serviceProvider = serviceProvider;
+            // AutomationDelegatingListView is defined in ServicesVisualStudio, which has
+            // InternalsVisibleTo this project. But, the markup compiler doesn't consider the IVT 
+            // relationship, so declaring the AutomationDelegatingListView in XAML would require 
+            // duplicating that type in this project. Declaring and setting it here avoids the 
+            // markup compiler completely, allowing us to reference the internal 
+            // AutomationDelegatingListView without issue.
+            var listview = new AutomationDelegatingListView();
+            listview.Name = "Options";
```

Comment:
Nah. It can probably be removed entirely, this was just a direct translation from xaml to c#